integration/tests/cook/test_basic.py

```diff
@@ -397,7 +398,8 @@ def memory_limit_exceeded_helper(self, command, executor_type):
         finally:
             util.kill_jobs(self.cook_url, [job_uuid])
 
-    @attr('memory_limit')
+    @pytest.mark.memlimit
+    @unittest.skipUnless(util.continuous_integration(), "Doesn't work in our local test environments")
```

Comment:
Do these tests currently get skipped?

Reply:
I always skip them locally because they *always* fail. I think they might only be passing in Travis due to some other constraint that exists in the Travis environment that we're not explicitly setting.